.changeset/red-snakes-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/connectivity': minor
+---
+
+[New Functionality] Add a `retry` option in the `DestinationFetchOption`. The retries apply to the destination by name call and not the full destination retrieval process. 
```

Comment:
> apply to the destination by name call and not the full destination retrieval process

can we rephrase this to make more clear what it means? Not sure if our users understand this sentence, I certainly have issues understanding what it means

Reply:
changed it to not mention technical details but when to enable it.